### PR TITLE
media-radio/unixcw: use HTTPS, update LICENSE, add missing braces

### PR DESCRIPTION
**Pull request by mm1ke**

### CODE REVIEW

**Comment by ConiKost** [comment]
No EAPI=8?

**Comment by mm1ke** [comment]
> No EAPI=8?

Good idea. I've updated the PR and added another commit only for the bump (since it's a stable package). Also made some changes around src_install (diff between r1 and r2):

```diff
4c4
< EAPI=7
---
> EAPI=8
14c14
< KEYWORDS="~alpha amd64 ppc x86"
---
> KEYWORDS="~alpha ~amd64 ~ppc ~x86"
24,25c24,25
< DEPEND="${RDEPEND}
< 	virtual/pkgconfig
---
> DEPEND="${RDEPEND}"
> BDEPEND="virtual/pkgconfig
49,51c49,50
< 	emake DESTDIR="${D}" install
< 	find "${D}" -name '*.la' -type f -delete || die
< 	dodoc ChangeLog NEWS README
---
> 	default
> 
60a60,61
> 
> 	find "${D}" -name '*.la' -type f -delete || die
```

File: media-radio/unixcw/unixcw-3.6.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	eapply -p0 "${FILESDIR}"/${PN}-3.6-tinfo.patch
+	# Bug# 837617 and 858278
+	sed -i -e "s/curses, initscr/ncurses, initscr/" \
+		-e "s/_curses_initscr/_ncurses_initscr/" configure.ac || die
+	eapply_user
+	eautoreconf
+}
+
+src_configure() {
+	econf --libdir=/usr/$(get_libdir) \
```
Comment (ConiKost) [inline media-radio/unixcw/unixcw-3.6.0-r2.ebuild:40]: Since you specify ```/usr``` manually, shouldn't be ```EPREFIX``` added?

Comment (mm1ke) [reply media-radio/unixcw/unixcw-3.6.0-r2.ebuild]: Correct, added. Thanks.

**Pull request closed without merging**